internal/pkg/service/buffer/store/model/model.go

```diff
@@ -40,6 +42,8 @@ type ExportBase struct {
 	ImportConditions ImportConditions `json:"importConditions" validate:"required"`
 }
 
+type Token storageapi.Token
```

Comment:
toto mi pride zbytocne, ked uz tak radsej alias `type Token = storageapi.Token`.

Tymto si zrusil vsetky metody, ktore tam su definovane, zbytocne sa to musi pretypovavat.